kupyna/tests/mod.rs

```diff
@@ -0,0 +1,5 @@
+mod kup256;
+mod kup48;
+
+mod kup384;
+mod kup512;
```

Comment:
This file is not needed. `cargo test` will automatically handle files in the tests folder.